js/src/personal/bryan/Bryan.module.css

```diff
@@ -1,7 +1,8 @@
 .title {
+    font-family: 'Inconsolata', monospace;
```

Comment:
nit: this should likely be in the theme object, unless you want this font just for the title